doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+- **Reference Implementation(RI)**: Builds on the requirements and specifications developed in RM, RAs and adds details so that it can be implemented.  Each Reference Architecture is expected to be implemented by at least one Reference Implementation.
+- **Reference Certification(RC)**: Builds on the requirements and specifications developed in the other documents and adds details on how an implementation will be verified, tested and certified. Both infrastructure verification and certification as well as VNFs/CNFs verifications and certifications will be covered.
+
+Below is a diagram of the different artifacts that will need to be created to support the implementation of the abstract concepts presented in the **Reference Model**, which are then applied to create the **Reference Architecture**, that will be deployed using the requirements spelled out in the **Reference Implementation**.
+
+<p align="center"><img src="./figures/tech_scope_3.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 2:</b> Description of the possible different levels of CNTT artefacts</p>
+
+<a name="3.0"></a>
+## 3. Principles
+
+Any specification work created within CNTT **must** obey to set of principles specified by CNTT:
+
+* [Reference Model Principles](../ref_model/chapters/chapter01.md#13-principles)
+* [Reference Architecture Principles](../ref_arch#principles)
+
+<a name="4.0"></a>
+## 4. Technology Direction
+
+<a name="4.1"></a>
+### 4.1 IO Virtualization
+
+There are different ways of which IO devices (such as NICs) are presented to workloads for consumption by those workloads. Here is a list of current methods of existing IO Virtualization:
+
+- Para-Virtualization method (software only).
+- Direct assignment with Virtual Technology.
+- Device Sharing with SR-IOV & Direct Assignment.
+- Para-Virtualization method (Hardware support).
+
+**Figure 3** below shows some of the relevant IO Virtualization techniques.
+
+<p align="center"><img src="./figures/tech_iov.png" alt="scope" title="Document Types" width="70%"/></p>
+<p align="center"><b>Figure 3:</b> Relevant IO Virtualization Techniques</p>
+
+<a name="4.1.1"></a>
+#### 4.1.1 Para-virtualization method (software only)
+
+This is the preferred method of IO virtualization as it provides flexibility and full abstraction of workloads from the underlying infrastructure. It usually relies on standard IO interfaces that are implemented in software.
+For Networking, there are two common networking interfaces used: virtio-net for KVM/QEMU and VMXNET for VMware.
```

Comment:
Is VMXNET proprietary? We haven't defined the qualification criteria to complement our stated principle/prohibition, which opens the door to questions such as:
- If the driver is open source, is that acceptable?
- If the driver is included in base Linux distributions, is that acceptable?
- If both of the above?
- If the ABI is published, as it is for venerable chips such as the 16550, is that acceptable?
- If a base driver/abstraction is embedded **"in"** the hardware via an on-board UEFI plugin, is that acceptable?
It's gets deep. Maybe we discuss F2F in Prague? // Note: This is not a request to block a merge for Prague, just a suggested discussion at Prague.

Reply:
This section @markshostak is not setting our policies, it is simply for explanation of those technologies to avoid any ambiguity.